.github/workflows/_build.yaml

```diff
@@ -140,3 +140,14 @@ jobs:
           ${{ env.PACKAGE_PATH }}/bin/bashparser
         if-no-files-found: error
         retention-days: 7
+
+  # For now only use the hard-coded artifact name.
```

Comment:
We don't need to hard-code it. You should be able to use:
```
artifact-${{ env.ARTIFACT_OS }}-python-${{ env.ARTIFACT_PYTHON }}
```
Also, please add a comment about the purpose of the following job.

Reply:
I tried using this but GitHub Actions raises an error - https://github.com/oracle-samples/macaron/actions/runs/4432579381

Reply:
> I tried using this but GitHub Actions raises an error - https://github.com/oracle-samples/macaron/actions/runs/4432579381

Yeah, looks like `env` context still can only be used at `step` level: https://github.com/actions/runner/issues/2394
Please use the fixed image name and add a TODO to use `env` when/if it is supported here.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/d2fe70dc8bc12b52071d18e885bf63b86f9dcc25